.agents/skills/README.md

```
@@ -62,13 +79,16 @@ To add a new skill:
 5. Update this README
 6. Test the skill workflow
 
-**Directory structure**:
+**Directory structure** (skills live in `.agents/skills/`; `.claude/skills` is a
+symlink to it for portability across agent runtimes):
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)